lib/elixir/lib/tuple.ex

```diff
@@ -1,6 +1,32 @@
 defmodule Tuple do
   @moduledoc """
   Functions for working with tuples.
+  
+  Please note the following functions for tuples are found in `Kernel`:
+  
+    * `elem/2` - for accessing a tuple by index
+    * `put_elem/3`
+    * `tuple_size/3`
```

Comment:
Why only have the "shortdoc" for just one of them? I think we should either do that for all or for none.